x-pack/plugins/apm/public/components/app/ServiceMap/cytoscapeOptions.ts

```diff
@@ -17,7 +17,8 @@ export const getSeverityColor = (nodeSeverity: string) => {
   switch (nodeSeverity) {
     case severity.warning:
       return theme.euiColorVis0;
-    case severity.minor || severity.major:
+    case severity.minor:
+    case severity.major:
```

Comment:
Unit test for `getSeverityColor`?

Reply:
It's private to these settings, and I didn't want to export it just for the purposes of testing. It is exposed (and was broken) in the storybook: 

![image](https://user-images.githubusercontent.com/9912/81621779-e8f52e00-93b4-11ea-9f28-e73dc955eb22.png)

I'd be open to exporting it just to add unit tests though.